doc/zstd_compression_format.md

```diff
@@ -381,7 +380,7 @@ There are 4 block types :
   This value cannot be used with current version of this specification.
 
 Block sizes must respect a few rules :
-- In compressed mode, compressed size if always strictly `< decompressed size`.
+- In compressed mode, `compressed size` is always strictly `< decompressed size`.
```

Comment:
IMO it should be `compressed size < decompressed size` or not in backticks, and the three bullet points should be consistent.